.github/workflows/pr_compliance.yml

```diff
@@ -0,0 +1,21 @@
+name: PR Compliance
+
+on:
+  pull_request_target:
+    types: [opened, edited, reopened]
+
+# Action should have write permission to make updates to PR
+permissions:
+  pull-requests: write
+
+jobs:
+  pr-compliance:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: mtfoley/pr-compliance-action@main
+        with:
+          ignore-authors:
+            - dependabot
+            - dependabot[bot]
+            - all-contributors[bot]
+          title-check-enable: false
```

Comment:
If this bot is able to do commitlint for us, why not let it, and remove the separate commitlint action? Less configuration is nice.

Reply:
Ping @fuadop - is this still something you have time for?

Reply:
Hey @JoshuaKGoldberg, I didn't get the notification for this all this while, Let me set this to true then. It makes sense to let just one GitHub action to both.